server/data/incidentTypeConfiguration/types.ts

```diff
@@ -38,3 +40,13 @@ export interface AnswerConfiguration {
   commentRequired: boolean
   nextQuestionId: string | null
 }
+
+/** Report type prisoner role configuration */
+interface PrisonerRoleConfiguration {
+  /** Role type for this report type */
+  prisonerRole: PrisonerInvolvementRole
+  /** If only one prisoner can have this role in a given report */
+  onlyOneAllowed: boolean
+  /** Indicates this role is active */
+  active: boolean
```

Comment:
not sure we'd wanna keep `active` here long term cos if a role is not allowed, we can just not list it inside `IncidentTypeConfiguration.prisonerRoles `

Reply:
Possible, not sure, but I think it makes sense to keep it at the moment as we're effectively translating the NOMIS config and the alternative is to remove these inactive roles altogether which is probably a step too far.